firefox_puppeteer/ui/toolbars.py

```diff
@@ -55,26 +71,32 @@ def menu_button(self):
         return self.marionette.find_element(By.ID, 'PanelUI-menu-button')
 
 
-class LocationBar(BaseLib):
+class LocationBar(UIBaseLib):
     """Provides access to and methods for the DOM elements contained in the
     locationbar (the text area of the ui that typically displays the current url)."""
 
-    dtds = ["chrome://branding/locale/brand.dtd",
-            "chrome://browser/locale/browser.dtd"]
-
-    def __init__(self, *args, **kwargs):
-        BaseLib.__init__(self, *args, **kwargs)
+    def __init__(self, marionette_getter, window):
+        UIBaseLib.__init__(self, marionette_getter, window)
         # TODO: A "utility" module that sets up the client directly would be
         # useful here.
+        self._autocomplete_results = None
+        self._identity_popup = None
+        
+		
         self.l10n = L10n(self.get_marionette)
         self.keys = Keys(self.get_marionette)
```

Comment:
Both class instances are not needed anymore. You can remove them in favor of `self.window.send_shortcut`.